content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -66,9 +66,11 @@ spec:
   - name: v1
     served: true
     storage: false
+  # The conversion section is introduced in kubernetes 1.13+ with a default value of
+  # None conversion (strategy sub-field set to None).
   conversion:
-    # a None strategy will assume same schema for all versions and only set apiVersion
-    # field of the CRs to the proper value on conversion.
+    # None conversion assumes the same schema for all versions and only set apiVersion
+    # field of custom resources to the proper value
```

Comment:
None conversion assumes the same schema for all versions and only sets the apiVersion